packages/circuit-ui/components/RadioButton/RadioButton.tsx

```diff
@@ -255,11 +273,23 @@ export const RadioButton = forwardRef(
         />
         <RadioButtonLabel
           htmlFor={id}
-          invalid={invalid}
+          invalid={invalid ?? false}
           className={className}
           style={style}
+          aria-describedby={helperText ? `${id}-helper` : undefined}
         >
           {label}
+          {helperText && (
```

Comment:
There are two conflicting requirements for the description:

1. The description should be interactive, which can be achieved by including it in the `label`.
2. The description should not be part of the accessible name (aka label) of the input. 

We solved this in the Toggle component by rendering the description once inside and once outside the `label` and hiding it from the accessibility tree and visually respectively. Please use a similar approach here.